backend/jest-dynamodb-config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = async () => {
+    const serverless = new (require('serverless'))();
+
+    await serverless.init();
+    const service = await serverless.variables.populateService();
+    const resources = service.resources[0].Resources;
+
+    console.log('DynamoDB Tables Mocked for tests', resources);
+
+    const tables = Object.keys(resources)
+        .map((name) => resources[name])
+        .filter((r) => r.Type === 'AWS::DynamoDB::Table')
+        .map((r) => r.Properties);
```

Comment:
No need to put a single arg between brackets

Reply:
@mourtisma our current ESLine rules force parenthesis on single arguments (my VScode even autofixes them back if I remove parenthesis)